justfile

```diff
@@ -61,4 +61,50 @@ make-tn-models project_url api_key endpoint='/api/users/':
     echo "Please run 'tn install-bun' to install bunx first."
     exit 1
   fi
-  bunx @thinknimble/tnm-cli read '{{project_url}}/api/schema/?format=yaml' -t '{{api_key}}' -u '{{endpoint}}'
\ No newline at end of file
+  bunx @thinknimble/tnm-cli read '{{project_url}}/api/schema/?format=yaml' -t '{{api_key}}' -u '{{endpoint}}'
```

Comment:
The endpoint can also just be the physical file if someone has it downloaded, but I most likely folks will use the link